datus/agent/node/explore_agentic_node.py

```
@@ -0,0 +1,333 @@
+                messages=f"{self.get_node_name()} exploration completed successfully",
+                input_data=user_input.model_dump(),
+                output_data=result.model_dump(),
+                status=ActionStatus.SUCCESS,
+            )
+            action_history_manager.add_action(final_action)
+            yield final_action
+
+        except ExecutionInterrupted:
+            raise
+
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)